recipes/python-poppler-qt5/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "python-poppler-qt5" %}
+{% set version = "21.3.0" %}
+
+# Note: python-poppler-qt5 versions follow Poppler versions. However,
+# poppler is pinned in conda-forge
+# (https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml).
+# We just build the binding with a higher version of Poppler, which is fine
+# -- it just exposes the API subset that was already in the older Poppler version
+# corresponding to the python-poppler-qt5 version.
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b477e8841f0ea0e09fdab6f2f305cf38df977d9e142e5693a374605d75dd6fe0
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - sip
+    - setuptools
+    - pyqt-builder
+    - pyqt
+    - poppler-qt {{ poppler }}
+  host:
+    - {{ compiler('cxx') }}
+    - python
```

We don't need all these host dependencies in build, and we shouldn't have the compiler in host.

```suggestion
requirements:
  build:
    - {{ compiler('cxx') }}
  host:
    - python
```

---

> When poppler-qt is removed from build requirements, its pkg-config (.pc) file is not found under `$CONDA_PREFIX/lib/pkgconfig`, but it is found under `$PKG_CONFIG_PREFIX_2/lib/pkgconfig`.

For basically everything that's not cross-compilation, you should just use `$PREFIX`, not `$CONDA_PREFIX` (which is an alias for `$BUILD_PREFIX`)